src/vcConvert.cpp

```diff
@@ -322,8 +322,11 @@ void vcConvert_ShowUI(vcState *pProgramState)
   // Convert Jobs --------------------------------
   ImGui::Columns(2);
 
+  const ImVec2 &size = ImGui::GetWindowSize();
   if (ImGui::IsWindowAppearing())
-    ImGui::SetColumnWidth(0, 300);
+    ImGui::SetColumnWidth(0, size.x * pProgramState->settings.presentation.convertLeftPanelPercentage);
+  else
+    pProgramState->settings.presentation.convertLeftPanelPercentage = ImGui::GetColumnWidth(0) / size.x;
```

Comment:
Is it possible to have a window size of `0`?

Reply:
Probably not for the value, but the left part could be dragged too small to recover, I'll set a minimum width for it.

Reply:
And setting maxinum width too.